hugolib/site.go

```diff
@@ -1251,7 +1253,7 @@ func (s *Site) RenderHomePage() error {
 
 	if !viper.GetBool("DisableRSS") {
 		// XML Feed
-		n.URL = s.permalinkStr("index.xml")
+		n.URL = s.permalinkStr(s.Info.RSSUri)
```

Comment:
I would just use viper.GetString("RSSUri") directly here and below .

I would expect this to work.